### PR TITLE
Feature/dashboard perils

### PR DESCRIPTION
**Pull request by hedvigoscar**

### CODE REVIEW

File: owldroid/src/main/java/com/hedvig/android/owldroid/ui/dashboard/DashboardFragment.kt [hunk 1]
```diff
@@ -65,46 +73,167 @@ class DashboardFragment : Fragment() {
...
-            )
+        })
+
+        actionContainer.show()
+    }
+
+    private fun makePerilCategoryRow(category: DashboardQuery.PerilCategory): PerilCategoryView {
+        val categoryView = PerilCategoryView(requireContext())
+        categoryView.categoryIconId = category.iconUrl()
+        categoryView.layoutParams = ViewGroup.MarginLayoutParams(
```
Comment (FredrikAreschoug) [inline owldroid/src/main/java/com/hedvig/android/owldroid/ui/dashboard/DashboardFragment.kt]: Just to make the fragment a bit smaller, we could put this as an initializer/builder function `PerilCategoryView`

File: owldroid/src/main/java/com/hedvig/android/owldroid/ui/dashboard/DashboardFragment.kt [hunk 2]
```diff
@@ -65,46 +73,167 @@ class DashboardFragment : Fragment() {
...
+            accumulator += perilTotalWidth
+            counter += 1
+        }
+
+        return counter
+    }
+
+    private fun makePeril(peril: DashboardQuery.Peril, subject: DashboardQuery.PerilCategory): PerilView {
+        val perilView = PerilView(requireContext())
+        perilView.layoutParams = ViewGroup.MarginLayoutParams(
```
Comment (FredrikAreschoug) [inline owldroid/src/main/java/com/hedvig/android/owldroid/ui/dashboard/DashboardFragment.kt]: Same suggestion as the initializer/builder function before

File: owldroid/src/main/java/com/hedvig/android/owldroid/ui/dashboard/DashboardFragment.kt [hunk 3]
```diff
@@ -65,46 +73,167 @@ class DashboardFragment : Fragment() {
...
+        additionalInformation.subtitle = resources.getString(R.string.DASHBOARD_MORE_INFO_SUBTITLE)
+
+        additionalInformation.layoutParams = ViewGroup.MarginLayoutParams(
+            ViewGroup.LayoutParams.MATCH_PARENT,
+            ViewGroup.LayoutParams.WRAP_CONTENT
+        ).also { lp ->
+            lp.topMargin = halfMargin
+            lp.marginStart = tripleMargin
+            lp.marginEnd = tripleMargin
+            lp.bottomMargin = tripleMargin
```
Comment (FredrikAreschoug) [inline owldroid/src/main/java/com/hedvig/android/owldroid/ui/dashboard/DashboardFragment.kt]: I now see that we actually set the margin diffrently. Builder wit default values?

Comment (hedvigoscar) [reply owldroid/src/main/java/com/hedvig/android/owldroid/ui/dashboard/DashboardFragment.kt]: This was actually in error, good catch ✅

**Pull request merged**